apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/AbstractClaimRdaSink.java

```diff
@@ -354,6 +356,8 @@ private void mergeBatch(long maxSeq, Iterable<RdaChange<TClaim>> changes) {
         entityManager.getTransaction().rollback();
       }
       entityManager.clear();
+      timerContext.stop();
```

Comment:
wouldn't the timer be better served to live in a finally block?

Reply:
We already have a finally that is cleaning up the transaction and I wasn't sure we'd really want to record the time if the commit or rollback threw an exception so I just added the stop in the finally block after that completed.

Reply:
ok, sounds good..thx.